content/pages/constitution.md

```diff
@@ -211,19 +211,44 @@ Committee office, M3 1003.
     2. The current committee will then select evaluate the applications
        received and select applicants for interviews;
 
-    3. The current committee will select interviewers. Interviewers should be
-       chosen by the current committee, and are expected to disclose any
+    3. A single interview shall be held for each candidate. Interviews shall
+       not exceed 30 minutes in length;
+
+    4. The current committee will select interviewers. Two interviewers should
+       be chosen by the current committee, and are expected to disclose any
        conflicts of interest when presented with the list of candidates. If a
        conflict of interest arises, an alternate interviewer shall be chosen.
        In any case, if no interviewer is explicitly chosen and there is no
-       conflict of interest, the current Chair shall conduct the interviews;
+       conflict of interest, the current Chair and Secretary shall conduct the
+       interviews;
+
+    5. Two interviewers must be present at every interview, and each shall
```

Comment:
If we have set out requirements, questions and qualifications beforehand, I think having two people do interviews (even if they are doing them separately to accommodate time and scheduling), is going to be a pretty consistent evaluation. Putting a requirement to have two people in every interview might just encourage having lots of different people to do interviews just so that we can make sure to have two people in every one. Alternatively it will make it a lot harder to schedule interviews.